contracts/interfaces/IDkg.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+interface IDkg {
+    function get_PK() external view returns (uint256, uint256);
+
+    function get_PK_for(uint participant_id) external view returns(uint, uint);
+
+    function get_committee_id_from_address(address addr) external view returns (uint);
+
+    function n_comm() external view returns (uint);
+}
```

Comment:
git shows a warning about no final newline, here and in a few other files.
(Worth setting your editor to always add this.)

Reply:
Fixed